common/helpers/field.js

```diff
@@ -68,7 +63,7 @@ function renderConditionalFields([key, field], index, obj) {
         const fieldName = item.conditional
         const field = fields[fieldName]
 
-        if (!field) {
+        if (!has(field, 'component') || !field.component) {
```

Comment:
Do you mean something like
`if (field.component === undefined  || field.component === null)`

Or it is intentional to include every falsey? 🤔